EventSubscriber/DeprecatedRouteSubscriber.php

```diff
@@ -49,19 +50,25 @@ public function onKernelResponse(ResponseEvent $event) {
     }
     $response = $event->getResponse();
     // set date in the deprecation response header
-    $deprecationDate = new \DateTime($deprecatedSince);
+    $deprecationDate = \DateTime::createFromFormat(DATE_ATOM, $deprecatedSince);
+    if(false === $deprecationDate){
+      throw new \Exception(self::DATE_FORMAT_ERROR_MESSAGE);
+    }
     $response->headers->set(self::DEPRECATION_HEADER, $deprecationDate->format(self::HTTP_DATE_FORMAT));
 
     // check to see if onKernelController set a sunset attribute
     if (!$deprecatedUntil = $event->getRequest()->attributes->get(self::SUNSET_ATTRIBUTE)) {
       return;
     }
     // set date in the sunset response header
-    $sunsetDate = new \DateTime($deprecatedUntil);
+    $sunsetDate = \DateTime::createFromFormat(DATE_ATOM, $deprecatedUntil);
```

Comment:
Should use DateTimeInterface::ATOM rather than DATE_ATOM

Reply:
fixed!